test/ParsedownTest.php

```diff
@@ -139,7 +139,7 @@ public function test_no_markup()
 
     public function testLateStaticBinding()
     {
-        include 'test/TestParsedown.php';
+        include dirname(__file__) . '/TestParsedown.php';
```

Comment:
I think we could just use `__DIR__` instead of `dirname(__FILE__)` as Parsedown no longer supports PHP 5.2.